e2e/src/utils/utils.js

```diff
@@ -424,19 +428,45 @@ export async function completeProtectWalletScreen() {
  *
  * @param senderPrivateKey: private key for wallet with funds
  * @param recipientAddress: wallet to receive funds
- * @param stableToken: ContractKit-recognized stable token
+ * @param stableToken: recognised token symbol (e.g. 'cUSD')
  * @param amountEther: amount in "ethers" (as opposed to wei)
  */
 export async function fundWallet(senderPrivateKey, recipientAddress, stableToken, amountEther) {
-  const kit = newKit(ALFAJORES_FORNO_URL)
-  const { address: senderAddress } = kit.web3.eth.accounts.privateKeyToAccount(senderPrivateKey)
+  const stableTokenSymbolToAddress = {
+    cUSD: '0x874069fa1eb16d44d622f2e0ca25eea172369bc1',
+  }
+  const tokenAddress = stableTokenSymbolToAddress[stableToken]
+  if (!tokenAddress) {
+    throw new Error(`Unsupported token symbol passed to fundWallet: ${stableToken}`)
+  }
+
   console.log(`Sending ${amountEther} ${stableToken} from ${senderAddress} to ${recipientAddress}`)
-  kit.connection.addAccount(senderPrivateKey)
-  const tokenContract = await kit.contracts.getStableToken(stableToken)
-  const amountWei = kit.web3.utils.toWei(amountEther, 'ether')
-  const receipt = await tokenContract
-    .transfer(recipientAddress, amountWei.toString())
-    .sendAndWaitForReceipt({ from: senderAddress })
+
+  const account = privateKeyToAccount(senderPrivateKey)
+  const senderAddress = account.address
+  const walletClient = createWalletClient({
+    account,
+    chain: celoAlfajores,
+    transport: http(ALFAJORES_FORNO_URL),
+  })
```

Comment:
nit: we can create a client that can do both wallet and public actions:

```suggestion
import { createWalletClient, http, publicActions } from 'viem'
 
const client = createWalletClient({ 
  account,
  chain: celoAlfajores,
  transport: http()
}).extend(publicActions) 
```